packages/office-addin-dev-settings/src/publish.ts

```diff
@@ -0,0 +1,18 @@
+import * as childProcess from "child_process";
+
+/* global console */
+
+export async function publish(zipPath: string): Promise<void> {
+  return new Promise((resolve, reject) => {
+    const command = `npx @microsoft/teamsfx-cli@0.11.0-rc.0 provision manifest --file-path ${zipPath}`;
```

Comment:
Given that this function is exported, can we add error checking for the `zipPath`?
Like seeing if it exists and checking if it is really a zip.

If the `teams-fx-cli` already does this, that's fine

Reply:
Good point.  While the "export" in this case is only internal to this package (you can't call it directly from outside the package) it would be good to make sure the zip file does exist.